api-gateway/src/main/resources/static/scripts/auth/reset-pwd-form/reset-pwd-form.controller.js

```diff
@@ -15,7 +15,12 @@ angular.module('resetPwdForm')
             })
             .catch(n => {
                 console.log(n)
-                $scope.errorMessages = n.data.message.split`\n`;
+                try {
```

Comment:
Could you explain what the difference is between the try and catch blocks here?

Reply:
It's because the return of an invalid password is "password":"message" and a normal message is "message":"message", So I am checking for both.